### DISCUSSION TITLE
Change of base currency not reflected in existing products

### DISCUSSION THREAD

**Discussion by maplesyrupman** [swellstores/community#231] [Help] [answered]
I had created some products while the base currency was set to USD. After changing it to CAD, new products that I create are priced in CAD, however existing products are still in USD. How do I remedy this?

**Comment by SusannaSwell** [comment, marked as answer]
hello @maplesyrupman ! you can clone existing product via Actions/Clone product. In this case CAD will be displayed as base currency for cloned product. and after review delete products in USD

**Reply by maplesyrupman** [reply]
Thanks Susanna.
